### PR TITLE
[HCD-215] Added doc for the clearAllData method

### PR DESCRIPTION
**Pull request by tom-win87**
Delete user data section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -623,6 +624,11 @@ Following an example:
         }
 ```
 
+## Delete user data
+Utilize the following method if an end-user requests to have their data deleted:
+```
```
Comment (carmelo-iriti) [inline README.md:629]: We could add the snippets for java and kotlin
```kotlin
    import com.sourcepoint.cmplibrary.util.clearAllData

    //...
    clearAllData(context)
    //...
```

```java
    import com.sourcepoint.cmplibrary.util.SpUtils;

    //...
    SpUtils.clearAllData(context)
    //...
```

**Comment by carmelo-iriti** [comment]
Hi @tom-win87 , 

do we have also the ticket number from jira? I'd like to connect the commits with the jira story

**Comment by tom-win87** [comment]
Hey Carmelo. Ticket number for reference is [HCD-215](https://sourcepoint.atlassian.net/jira/software/projects/HCD/boards/41?selectedIssue=HCD-215).

File: README.md (hunk 1 above)
Comment (tom-win87) [reply README.md:635]: Done

**Pull request merged**